extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -514,6 +530,21 @@ export class InstrumentationService {
     this.win_.setTimeout(this.win_.clearInterval.bind(this.win_, intervalId),
         maxTimerLength * 1000);
   }
+
+  /**
+   * Checks to confirm that a given trigger type is allowed for the element.
+   * Specifically, ti confirms that if the element is in the embed, only a
+   * subset of the trigger types are allowed.
+   * @param  {!AnalyticsEventType} triggerType
+   * @param  {!Element} element
+   * @return {boolean} True if the trigger is allowed. False otherwise.
+   */
+  isTriggerAllowed_(triggerType, element) {
+    if (element.ownerDocument.defaultView != this.win_) {
+      return (AllowedInEmbed.indexOf(triggerType) > -1);
+    }
+    return false;
```

Comment:
sorry. the PR is not ready yet. 
My local machine was borking on tests because of chrome 54 (which you fixed. thanks!) so i uploaded a commit. Will reply to the PR when it is ready for review.